webapp/styles/AccountSettings.module.scss

```diff
@@ -0,0 +1,66 @@
+@import 'Common.module.scss';
+
+.profileSettingForm {
```

Comment:
the css u got was too complicated. Very hard for another developer to track. I simplify it

Reply:
I fixed it